### DISCUSSION TITLE
Sturggling to integrate tRPC with Nextjs app router and clerk auth

### DISCUSSION THREAD

**Discussion by alexander-densley** [trpc/trpc#5503] [Q&A]
Title pretty much says it all, would love some resources for nextjs app router and clerk auth with trpc

**Comment by undercontr** [comment]
There is this page explains that we can use fetchRequestHandler for setting up for app router but it's not straight forward and have lot of configuration, i think we need to wait for createNextApiHandler equivalent for app router,

https://trpc.io/docs/server/adapters/nextjs#route-handlers
